pkg/client/collector_test.go

```diff
@@ -95,31 +101,6 @@ func TestCollectorFuzzSpans(t *testing.T) {
 	}
 }
 
-func TestCodec(t *testing.T) {
```

Comment:
when replacing codecs (ex squashing out a 3rd party dep like you have). I typically keep a test dep to ensure it is interoperable, at least for a while. That way you can tell if your endpoint works with zipkin or not (ex it accepts json or thrift, and your local codec can test against something else)